Add pull request template for Pyodide Github repository [skip ci] (#1925)
pyodide/pyodide: .github/PULL_REQUEST_TEMPLATE.md

> <!-- Thanks for contributing to Pyodide! Any improvements are welcome,
>      so don't be afraid of making a PR. To help make things go a bit more
>      smoothly we would appreciate that you go through this template. -->

Reviewer: ```suggestion
<!-- Thank you for contributing to Pyodide! All improvements are welcome,
     so don't be afraid to make a PR. -->
```

### Description

<!-- Please explain what your PR is about:
     - reasoning for the change
     - some details of updated code
     - any noteworthy choices to be aware of

>      If there is a relavant issues, please refer those by #<issue_id> -->

Reviewer: ```suggestion
	Please refer to any related issues by #<issue_id> -->
```
